Amplitude/Amplitude.m

```diff
@@ -1023,6 +1025,12 @@ - (void)makeEventUploadPostRequest:(NSString*) url events:(NSString*) events num
     [request setValue:@"application/x-www-form-urlencoded" forHTTPHeaderField:@"Content-Type"];
     [request setValue:[NSString stringWithFormat:@"%lu", (unsigned long)[postData length]] forHTTPHeaderField:@"Content-Length"];
 
+    if (_token != nil) {
+        NSString *auth = [NSString stringWithFormat:@"Bearer %@", _token];
+        AMPLITUDE_LOG(@"Attaching bearer");
```

Comment:
Do we wanna include the authToken together with this log?

Reply:
Sure let's